src/librustdoc/html/render/print_item.rs

```diff
@@ -37,7 +37,7 @@ use crate::html::markdown::{HeadingOffset, MarkdownSummaryLine};
 use crate::html::render::{document_full, document_item_info};
 use crate::html::url_parts_builder::UrlPartsBuilder;
 
-/// Generates a Rinja template struct for rendering items with common methods.
+/// Generates a Askama template struct for rendering items with common methods.
```

Comment:
Super-nitpicky nit 😸 
```suggestion
/// Generates an Askama template struct for rendering items with common methods.
```

Reply:
Good catch! :D